README.md

```diff
@@ -55,7 +55,21 @@ julia> uprint(pac, :block)
  ▀█████████████████ 
   ▀██████████████▀  
      ▀▀██████▀▀     
+
+julia> uprint(pac, :sextant)
+ 🬞🬵🬻██🬺🬱🬏
+🬞███🬕🬨███🬓
+🬻█████🬝🬆🬀
```

Comment:
Why is so much detail lost here ... Shouldn't it be bigger?

Reply:
This is what I see:
```
julia> uprint(pac, :sextant)
 🬞🬵🬻██🬺🬱🬏
🬞███🬕🬨███🬓
🬻█████🬝🬆🬀
█████🬲🬏  🬇
🬨██████🬺🬱🬏
 🬬██████🬝🬀
  🬁🬊🬎🬎🬆🬀
```
Well, my browser font doesn't seem to have sextants (my terminal font does), but it's six lines long, not just three.

Reply:
Ah it's just browser fonts

Reply:
![pac-sixel](https://github.com/user-attachments/assets/e5e19bb9-684a-465a-bd5a-950094e1bedb)

Reply:
Ah ok perfect!

Reply:
(I am using the [JuliaMono](https://juliamono.netlify.app) font.)